build/Makefile.lnx

```diff
@@ -0,0 +1,6 @@
+BINARY_SERVER=alm
+BINARY_CLIENT=alm-cli
+FIND=find
+DBHOST=localhost

```

Comment:
I'm in the middle of creating a better `docker-compose.yml` file to be able to link things a bit better.